### PR TITLE
postgresql_db: document when pg_restore is used

### PR DESCRIPTION
**Pull request by rosowiecki**
##### SUMMARY
Documentation when pg_restore is used.

Fixes: #588 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

postgresql_db

### CODE REVIEW

File: plugins/modules/database/postgresql/postgresql_db.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
     - The format of the backup will be detected based on the target name.
     - Supported compression formats for dump and restore include C(.pgc), C(.bz2), C(.gz) and C(.xz)
     - Supported formats for dump and restore include C(.sql) and C(.tar)
+    - Restore program is selected by target file format: C(.tar) and C(.pgc) are handled by pg_restore, other with pgsql.
```
Comment (Andersson007) [inline plugins/modules/database/postgresql/postgresql_db.py:70]:
```suggestion
    - "Restore program is selected by target file format: C(.tar) and C(.pgc) are handled by pg_restore, other with pgsql."
```

**Comment by rosowiecki** [comment]
Sorry: I definetely need my own python3 environment for sanity checks

**Comment by felixfontein** [comment]
@rosowiecki you can also use docker: `ansible-test sanity --docker -v --test <testtorun>`. It will then use exactly the same environment as in Shippable.

**Comment by Andersson007** [comment]
@rosowiecki thanks for the contribution!
@felixfontein thanks for reviewing!

**Comment by Andersson007** [comment]
merged https://github.com/ansible-collections/community.general/pull/589 into master

**Pull request merged**